recipe/meta.yaml

```diff
@@ -44,6 +45,7 @@ requirements:
     - pkgconfig
     - patch     # [unix]
     - m2-patch  # [win]
+    - llvm-openmp 14.0.6 # [osx]
```

Comment:
how come build as well as host?

Reply:
It has a strong run_exports. But now that I think of it, it will also apply if in host. I'll remove it from there.